examples/apps/ai-collab/src/components/TaskGroup.tsx

```diff
@@ -194,7 +198,8 @@ export function TaskGroup(props: {
 									);
 									const { originalBranch, forkBranch, forkView, newBranchTargetNode } =
 										branchManager.checkoutNewMergedBranchV2(
-											props.treeView,
+											// TODO: Remove cast when TreeViewAlpha becomes public
+											props.treeView as TreeViewAlpha<typeof SharedTreeAppState>,
```

Comment:
Could we add an `@alpha` type-narrowing function for this? Then we could do something like:
```typescript
assert(isTreeViewAlpha(props.treeView));
```

Reply:
Can't hurt. Since it always succeeds, I made it a "take and return" rather than a type guard, so that the user doesn't have to handle the else case.